terraform/modules/database/database.tf

```diff
@@ -9,6 +9,13 @@ resource "aws_rds_cluster_instance" "cluster_instances" {
   auto_minor_version_upgrade = "true"
   performance_insights_enabled = "true"
   preferred_maintenance_window = "${var.preferred_maintenance_window}"
+  tags = "${merge(
+    var.default_tags,
+    map(
+      "Name","upload",
+      "Env","${var.deployment_stage}"
```

Comment:
Add `Env` tag in the `default_tags` map instead of specifying it in every resource

Reply:
Since I added the default tags as a variable, i couldn't actually source a variable inside a variable when i am defining it :( I have to add it separately when i merge everything